packages/common/src/objects.ts

```diff
@@ -40,7 +40,7 @@ export const mapKeys = (obj: object, mapping: (key: string) => string) =>
             .map(([k, v]) => [mapping(k), v]))
 
 
-export function isPlainObject(value: unknown): value is object {
+export function isPlainObject(value: unknown): value is Record<string|number|symbol, any>{
```

Comment:
using `any` makes it weakly typed. better use `unknown`. typescript 4.9 will make working with such types nicer: https://devblogs.microsoft.com/typescript/announcing-typescript-4-9-beta/#in-narrowing